packages/devtools_app/lib/src/screens/memory/memory_controller.dart

```diff
@@ -304,6 +305,9 @@ class MemoryController extends DisposableController
     _memoryTracker?.stop();
     _memoryTrackerController.add(_memoryTracker);
 
+    tracingPaneController.dispose();
+    tracingPaneController = TracingPaneController();
```

Comment:
why does this controller need to be recreated and the other two (profilePaneController and diffPaneController) do not?

Reply:
good question that triggered nice refactoring. Thank you!